src/screens/FeedScreen/Content.js

```diff
@@ -391,7 +391,7 @@ export const styles = StyleSheet.create({
     width: 4,
     height: 4,
     borderRadius: 4,
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` seems fine as long as it aligns with the new design requirements and doesn't affect readability or accessibility. Please ensure that this color change is consistent across all similar components for a uniform user interface.
```diff
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->